solutions/src/main/java/ru/sberbank/school/task02/util/ArgumentParser.java

```diff
@@ -0,0 +1,58 @@
+package ru.sberbank.school.task02.util;
+
+import org.apache.commons.cli.*;
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ArgumentParser {
+
+    private final Options options;
+    private final CommandLineParser parser;
+    private final HelpFormatter helpFormatter;
+
+    public ArgumentParser() {
+
+        parser = new DefaultParser();
+        helpFormatter = new HelpFormatter();
+        options = new Options();
+    }
+
+    public ArgumentParser(Options options) {
+
+        this();
+        addOptions(options);
+    }
+
+    public Map<String, String> parseArgs(String[] args) {
+
+        try {
+
+            CommandLine cmd = parser.parse(options, args, true);
+            Map<String, String> parsedArgs = new HashMap<>();
+
+            for(Option option : options.getOptions()) {
```

Comment:
WhitespaceAround: &#39;for&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-985803638>*